### ISSUE TITLE
Quick shops items do not teleport users

### ISSUE THREAD

**Issue by Kirrasio** [myzticbean/QSFindItemAddOn#30]
Hi there, just updated quick shops and qs item find, when using the search feature and clicking the items, it does not teleport. No errors in console, just unresponsive buttons.

Minecraft version: Purpur 2176 -1.20.4
Quick shops version: QuickShop-Hikari 6.1.0.2
QSFindItemAddOn: 2.0.6.0

**Comment by rynstwrt** [comment]
I have this exact same issue, too. Even with `tp-player-directly-to-shop: true` in the config.

**Comment by rynstwrt** [comment]
I have this exact same issue, too. Even with tp-player-directly-to-shop: true in the config.

**Comment by myzticbean** [comment]
> I have this exact same issue, too. Even with tp-player-directly-to-shop: true in the config. Running version 2.0.6.0 and QuickShop-Hikari.

@rynstwrt can you attach the shop tp part of ur config here? As shown below:
```yaml
player-shop-teleportation:
  direct-shop-tp-mode:
    tp-player-directly-to-shop: true
    click-to-teleport-message: '&6&lClick to teleport to the shop!'
    shop-tp-no-permission-message: '&cYou don''t have permission to teleport to shop!'
    unsafe-shop-area-message: '&cThe area around the shop is unsafe!'
    tp-delay-in-seconds: 2
    tp-delay-message: '&6You will be teleported in &c{DELAY} &6seconds...'
    tp-to-own-shop-no-permission-message: '&cYou cannot teleport to your own shop!'
  nearest-warp-tp-mode:
    tp-player-to-nearest-warp: true
    do-not-tp-if-warp-locked: true
```

**Comment by myzticbean** [comment]
I found the issue, will be fixed in the next build.

**Comment by myzticbean** [comment]
@Kirrasio @rynstwrt till this fixed build gets uploaded to spigot/modrinth, you can download it from here: https://github.com/myzticbean/QSFindItemAddOn/releases/tag/v2.0.6.1-release

**Comment by myzticbean** [comment]
Fixed in v2.0.6.1
